frontend/app/views/spree/shared/_main_nav_bar.html.erb

```diff
@@ -1,55 +1,71 @@
-<% if spree_navigation_data.any? %>
+<% if spree_top_level_items_for_menu('spree-all-main').present? %>
```

Comment:
@MatthewKennedy this cannot be hardcoded - maybe let's also add `locale` column to `Menu` model and find that menus dynamically:

```
current_store.menus.for_locale(I18n.locale) || current_store.menus.first
```

Reply:
Sure, no problem

Reply:
actually, it should be 

```ruby
current_store.menus.for_locale(I18n.locale).for_footer || current_store.menus.for_footer.first
```

we should have a helper method

```
def spree_menu(kind = `header`)
  scope = current_store.menus.for_kind(kind)

  scope.for_locale(I18n.locale) || scope.first
end
```

Reply:
Noted!